### ISSUE TITLE
[form] validation cannot handle file input

### ISSUE THREAD

**Issue by prudho** [fomantic/Fomantic-UI#2712]
# Bug Report
Form validation actually cannot handle file input types. For example, I cannot trigger a validation error if no input file is provided.

As a quickfix, i changed the `selector.field` parameter from
```js
'input:not(.search):not([type="file"]):not([type="reset"]):not([type="button"]):not([type="submit"]), textarea, select'
```
to
```js
'input:not(.search):not([type="reset"]):not([type="button"]):not([type="submit"]), textarea, select'
```
but maybe there's other things to patch further (`selector.input` maybe ?).

## Steps to reproduce
Go to the related [JSFiddle](https://jsfiddle.net/hv4jemqb/) and try to validate the form without providing a file input.

## Expected result
The form shouldn't be validated, and therefore submited.

## Actual result
The form is validated since no validation is triggered for the input file.

## Testcase
[JSFiddle here](https://jsfiddle.net/hv4jemqb/).

## Version
2.9.2

**Comment by lubber-de** [comment]
🤔  We explicitely added file input exlusion by #1950 because jquery 3.4 bound events on such field which broke validation. 
Of course one can adjust the selector as you did

**Comment by lubber-de** [comment]
Mmh, after quicke test, the issue in #1950 does not seem to happen anymore when i just revert the changein the current codebase...although still using 3.4.1 / the same jsfiddle i used back then.... I'll investigate

Using
```
        selector: {
        	field: 'input:not(.search):not([type="reset"]):not([type="button"]):not([type="submit"]), textarea, select',
          input: 'input'
        },
```
Seems to work with your jsfiddle. I will also test invisible file inputs like in the jsfiddle back then

**Comment by lubber-de** [comment]
Fixed by #2713 Please review :)
See your adjusted jsfiddle here https://jsfiddle.net/lubber/tc9ox8dj/
